script/script.js

```diff
@@ -406,15 +406,27 @@ const usersSortFilter = {
             }
         },
         {
-            view: "button",
+            view: "threeStateButton",
             id: "sortAscBtn",
-            value: "Sort asc",
-            width: 150,
-            css: "webix_primary",
+            state: 0,
+            states: {
+                0: "Off",
```

Comment:
can't we define colors right here and apply inline-styling based on color value, rather then adding new css for each new state... Doesn't look dynamic. Absolutely wasted.